internal/cf/client.go

```diff
@@ -51,8 +56,10 @@ type clientCacheEntry struct {
 }
 
 var (
-	cacheMutex  = &sync.Mutex{}
-	clientCache = make(map[clientIdentifier]*clientCacheEntry)
+	cacheMutex        = &sync.Mutex{}
+	clientCache       = make(map[clientIdentifier]*clientCacheEntry)
+	cfCache           *facade.Cache
```

Comment:
use term "resourceCache" to have a clearer definition (and a better separation from clientCache)